### PR TITLE
Add diff tests for `scale` param in `dbStats`

### PR DESCRIPTION
**Pull request by rumyantseva**
See also https://github.com/FerretDB/FerretDB/pull/2418.

### CODE REVIEW

File: tests/diff/helpers.go [hunk 1]
```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package diff contains diff tests.
```
Comment (rumyantseva) [inline tests/diff/helpers.go:15]: Linter requires the description for the package.

**Review by chilagrow — APPROVED** [review]
LGTM

**Review by w84thesun — APPROVED** [review]

**Comment by AlekSi** [comment]
diff tests fail for MongoDB. @rumyantseva @b1ron PTAL

**Review by b1ron — APPROVED** [review]
So we removed `MaxFloat` and it worked.

Why?

**Review by chilagrow — APPROVED** [review]

**Comment by rumyantseva** [comment]
@chilagrow maxFloat +1 shouldn't cause an error either. There is an interesting way how `scale` works: 
```go
	case errors.Is(err, errLongExceededPositive):
		return math.MaxInt32, nil
```

I think I added the test for `MaxFloat` before I learned this interesting behavior.

**Comment by rumyantseva** [comment]
So, basically, if `scale` is too big, error is not returned but `math.MaxInt32` is used instead.

**Pull request merged**